src/components/shared/inputs/FormattedNumberInput.tsx

```diff
@@ -50,10 +51,10 @@ export default function FormattedNumberInput({
   ]
 
   useLayoutEffect(() => {
-    const accessory = document.getElementById(accessoryId)
-    if (!accessory) return
-    setAccessoryWidth(accessory.clientWidth)
-  }, [accessory, accessoryId])
+    const accessoryContainer = document.getElementById(accessoryId)
+    if (!accessoryContainer) return
+    setAccessoryWidth(accessoryContainer.clientWidth)
+  }, [accessoryId])
```

Comment:
Should not be referencing accessory prop, should reference container of accessory